doc-General_Configuration/topics/Configuration.adoc

```diff
@@ -2690,13 +2678,9 @@ The following is an example of regionalized database scenario:
 * Enable only the *Reporting*, *Scheduler*, and *User Interface Server* Roles.
 To perform database maintenance items, such as scheduled backups, on the top-level region (master), also enable the *Database Operations* role.
 * No additional settings aside from assigning the region ID. No need to configure any replication.
-. *Create Region Number 1*
-.. Add replication worker settings pointing to the VMDB for *Region 99*.
-.. Enable *Database Synchronization Server* role on one *Server* in the *Region*. If you have a second *Server* in the same region, do not enable the *DB Synchronization* role.
-Do not enable more than one *Database Synchronization Role* per Region.
-. *Create Region Number 2*
-.. Add replication worker settings pointing to the VMDB for *Region 99*.
-.. Enable *Database Synchronization Server* role on one *Server* in the *Region*. If you have a second *Server* in the same region, do not enable the *DB Synchronization* role. Do not enable more than one *Database Synchronization Role* per Region.
+. *Create Region Number 1*.
+. *Create Region Number 2*.
+
```

Comment:
Completely agree; I've removed the section. The section below it covers everything.